commands/polclear.js

```diff
@@ -0,0 +1,60 @@
+
+const Discord = require('discord.js');
+/**
+ * 
+ * @param {Number} server 
+ * @param {Rcon} rcon 
+ * @param {Discord.Message} msg 
+*/
+async function runcommand(server, rcon, msg, extra) {
```

Comment:
remove extra as argument it is not used as far is i can see

Reply:
it is not used in that function, it is used in an if before the function is run (check if extra commands, if so tell them bad boy.)
```suggestion
async function runcommand(server, rcon, msg) {
```